src/query/users/src/user_mgr.rs

```diff
@@ -133,6 +133,16 @@ impl UserApiProvider {
         }
     }
 
+    // Get the tenant all user nums.
+    #[async_backtrace::framed]
+    pub async fn get_user_nums(&self, tenant: &Tenant) -> Result<usize> {
```

Comment:
get_user_nums -> count_users? quq

Reply:
Delete this function. Directly call user_api.get_raw_users